src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -934,44 +967,71 @@ private void DrawPackage(INugetPackage package, GUIStyle packageStyle, GUIStyle
                 }
 
                 GUILayout.FlexibleSpace();
-                if (installed != null && installed.PackageVersion != package.PackageVersion)
-                {
-                    GUILayout.Label($"Current Version {installed.PackageVersion.FullVersion}");
-                }
 
-                if (package.Versions.Count <= 1)
+                // Show the installed version of the package if it exists
+                if (installed != null)
                 {
-                    GUILayout.Label($"Version {package.PackageVersion.FullVersion}");
+                    GUILayout.Label($"Installed Version: {installed.PackageVersion.FullVersion}");
                 }
-                else
+
+                // Show the version selection dropdown only on Updates tab OR on Online tab if the package is not installed and not already in Unity
```

Comment:
Shouldn't it be possible to update a package from the "Online tab" or even from the "Installed tab"?
E.g. in Visual Studio you can update / uninstall package from all 3 tabs: 
Visual Studio Installed tab:
![image](https://github.com/GlitchEnzo/NuGetForUnity/assets/53140583/57ee1dea-bc64-48a0-9f91-a032cd9ac17a)
Visual Studio Online tab:
![image](https://github.com/GlitchEnzo/NuGetForUnity/assets/53140583/77f18d7f-8f6f-4449-b4dc-cbee6be5783e)
Visual Studio Update tab:
![image](https://github.com/GlitchEnzo/NuGetForUnity/assets/53140583/b6a59c52-0412-4743-aefe-a1c5e5ed1257)

Reply:
The idea behind this was to reduce the number of buttons, as the UI as it currently is gets a bit too squashed already (I can see that VS's Nuget has a separate menu for these types of package manipulation), and at the same time it seems more intuitive to look to `Installed` tab if you want to uninstall, `Online` tab if you're looking for a new package and `Updates` tab if you're looking to get some other version of an already installed package.
 I will update the screenshots accordingly when we settle on the design.

Reply:
We noticed buttons with label about the version next to them often start to overlap with package ID from the left. 

Also we decided to add Show downgrades check box for a couple of reasons. First we assumed that we want to expose the downgrade functionality and definitely have it on updates tab. Before this commit only newer versions were fetched for both V2 and V3 API and we changed V3 to includeAllVersions. Since V2 has no such easy API we left it working as always.

When we added that we realized that now updates tab almost always shows all the installed packages some actually having new versions but all of them having some old versions and most often we go to that tab to easily see what packages actually have newer versions. Also we often actually want to use update all button to update all the packages that have new version. If we didn't hide downgrade options it might look like that button would update some packages but downgrade others since it would actually update to the versions you selected next to each package.

This seemed as a confusing UX so we hoped to make it more streamlined for the most often needed actions while still giving user options to do advanced stuff.

Reply:
Ok I understand. In Visual Studio the Updates tab can also only update. They use the Installed tab to downgrade. And with the amount of text / Bottons inside a single row I also noticed this so I agree with the splitting.